cocos/renderer/CCTextureCache.h

```diff
@@ -115,6 +115,14 @@ class CC_DLL TextureCache : public Ref
     */
     virtual void addImageAsync(const std::string &filepath, const std::function<void(Texture2D*)>& callback);
 
+    /** Init a given Texture2D object with a file image.
+    * It is much the same as addImageAsync, except an existing texture is provided.
+     @param texture An existing texture.
+     @param filepath A null terminated string.
+     @param callback A callback function would be invoked after the image is loaded.
+    */
+    virtual void initImageAsync(Texture2D* texture, const std::string& filepath, const std::function<void(Texture2D*)>& callback);
```

Comment:
This interface is strange. 
`TextureCache::addImageAsync` is used for parsing image file to texture asynchronously, but if you have gotten a texture and you don't want to add it to `TextureCache`, why to add this interface to `TextureCache`? 
Is it ok to just use your `Texture2D*` instance directly?

Reply:
Just because I want to use the async queue in textureCache.

Reply:
How about using `Scheduler::performFunctionInCocosThread` or `Scheduler::schedule` with repeat once to achieve this functionality?